xdsl/dialects/csl/csl_stencil.py

```diff
@@ -206,7 +206,8 @@ class ApplyOp(IRDLOperation):
 
     accumulator = operand_def(AnyTensorTypeConstr | AnyMemRefTypeConstr)
 
-    args = var_operand_def(Attribute)
+    args_rchunk = var_operand_def(Attribute)
+    args_dexchng = var_operand_def(Attribute)
     dest = var_operand_def(stencil.FieldTypeConstr | AnyMemRefTypeConstr)
```

Comment:
This is the actual change.

Reply:
This is mostly for cleaning up the dialect. Effectively, this simplifies a number of places in various code files, and needed changes to `operandSegmentSizes` in a number of mlir filechecks.